### PR TITLE
Re-enable + fix color quality test

### PR DESCRIPTION
**Pull request by madelynkasula**

### CODE REVIEW

File: test/unit/qualityTest.test.js [hunk 1]
```diff
@@ -174,30 +174,48 @@ describe('Model quality test', () => {
...
-  //     expect(result.recall).toBeGreaterThanOrEqual(0.5);
-  //   }
-  // });
+  test('Color test', async () => {
+    const trainSize = TRAIN_SIZE;
+
+    const idsByColor = {
+      red: [6, 7],
+      green: [2, 3],
+      blue: [4, 5]
```
Comment (madelynkasula) [inline test/unit/qualityTest.test.js]: @uponthesun do you think i should add more colors here? downside: it makes the test take longer; upside: ?

File: test/unit/qualityTest.test.js [hunk 2]
```diff
@@ -174,30 +174,48 @@ describe('Model quality test', () => {
...
+    };
+
+    const colorMatch = (rgb, colorIds) => {
+      const rgbs = Object.values(fishData.colors).map(color => color.rgb);
+
+      let match = false;
+      colorIds.forEach(id => {
```
Comment (uponthesun) [inline test/unit/qualityTest.test.js]: I find the inner `return` statement from this block confusing, can we use the `for(const id of colorIds) {}` syntax instead so it just sets the `match` variable normally and then `break`s? Feel free to push back if you don't like that 😁

File: test/unit/qualityTest.test.js (hunk 1 above)
Comment (uponthesun) [reply test/unit/qualityTest.test.js]: I think this test should specifically be for the short words mode, so we only need to include the ones listed there. So I think this is fine.

File: test/unit/qualityTest.test.js (hunk 2 above)
Comment (madelynkasula) [reply test/unit/qualityTest.test.js]: ah, yeah, that would read better. i'll update!

**Pull request merged**